keycloak/kustomization.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+    app.kubernetes.io/name: keycloak
+resources:
+  - deployment.yaml
+  - service.yaml
+  - service-account.yaml
+  - ingress.yaml
+configMapGenerator:
+  - name: keycloak-config
+    literals:
+      - KC_HOSTNAME_URL=https://keycloak.hashbang.sh
+      - KC_DB=postgres
+      - KC_DB_URL_HOST=userdb-attempt-too-do-user-989073-0.db.ondigitalocean.com
```

Comment:
I don't love that we're reusing this DB

Reply:
We're using this every service we currently have. I agree it's not ideal but DBs aren't cheap.

Reply:
> I don't love that we're reusing this DB

We are separating it into a separate database on the instance, though. I think that is fine.